pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/MultiStageBrokerRequestHandler.java

```diff
@@ -165,7 +165,8 @@ private BrokerResponseNative handleRequest(long requestId, String query,
 
     ResultTable queryResults;
     try {
-      queryResults = _queryDispatcher.submitAndReduce(requestId, queryPlan, _mailboxService, queryTimeoutMs);
+      queryResults = _queryDispatcher.submitAndReduce(requestId, queryPlan, _mailboxService, queryTimeoutMs,
+          sqlNodeAndOptions.getOptions());
```

Comment:
nit: please use pinot intellij checkstyle so that it doesn't automatically reformat code like these changes

Reply:
Done